float32 dn_star
float32 df_star
float32 focal_star
geometry_msgs/Point[] targets_im_up_star
geometry_msgs/Point[] targets_im_down_star
float32[] targets_d_star
geometry_msgs/Quaternion[] targets_orientation_star
cinempc/Weights weights
